stackv4/unikernel.ml

```diff
@@ -13,33 +13,38 @@ module Main (C:CONSOLE) (S:STACKV4) = struct
 
   let start console s =
 
-    C.log_s console
-      (sprintf "IP address: %s\n"
-        (String.concat ", " (List.map Ipaddr.V4.to_string (S.IPV4.get_ip (S.ipv4 s)))))
-    >>= fun () ->
+    let ips = List.map Ipaddr.V4.to_string (S.IPV4.get_ip (S.ipv4 s)) in
+    C.log_s console (sprintf "IP address: %s\n" (String.concat ", " ips))
 
-    S.listen_udpv4 s 53 (
+    >>= fun () ->
+    let loc_port = 53 in
+    S.listen_udpv4 s loc_port (
       fun ~src ~dst ~src_port buf ->
-        C.log_s console "got udp on 53"
+        C.log_s console
+          (red "UDP %s.%d > %s.%d: \"%s\""
```

Comment:
Well, the dot is weird, because it's already the separator inside ipv4 addresses...